lib/src/hidden_ranges/line_numbering_breakpoint.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:equatable/equatable.dart';
+
+class LineNumberingBreakpoint with EquatableMixin {
```

Comment:
Suggest to comment purpose of this class

Reply:
`+`